### ISSUE TITLE
Symfony deprecated Intl::getRegionBundle() causes PHP Fatal error:  Allowed memory size...

### ISSUE THREAD

**Issue by nils-m** [Sylius/Sylius#10654]
| Q                | A
| ---------------- | -----
| Bug report?      | no
| Feature request? | no
| BC Break report? | no/yes
| RFC?             | no/yes
| Sylius version   | 1.5.3

![image](https://user-images.githubusercontent.com/666276/64177245-64363580-ce5f-11e9-8696-bf302827e58b.png)

When trying to edit a zone with 28 Countries (EU ;-))
Got a blank page and a PHP Fatal error allowed memory size of X exhausted in the log.
I set memory_limit to -1 (no limit) 
![image](https://user-images.githubusercontent.com/666276/64178120-276b3e00-ce61-11e9-816e-f54319b19756.png)

Editing _Sylius\Component\Addressing\Model\Country_ fixed the issue. Do I submit a patch ? on master or 1.5 ?

```PHP
// Sylius\Component\Addressing\Model\Country
....
// use Symfony\Component\Intl\Intl; removed
use Symfony\Component\Intl\Countries;
...
// return Intl::getRegionBundle()->getCountryName($this->code, $locale); removed
return Countries::getName($this->code, $locale);
```

**Comment by evanesis** [comment]
Hello ! 

I spent some time on this exact problem yesterday afternoon. 

With a much much bigger zone members list (let's say something called "Rest of the world"), replacing the old `Intl::getXXXBundle()` API reduces drastically the time used, but then it's still a bit too much time consuming (went from 30s timeout to a 15s loading time on a 150+ Country members zone).

The problem is, when your members are `Country` type for example : 
- Each `CountryCodeChoiceType` has a list of `Country` elements as choices 
- Each displayed value is then rendered by using `Country->getName()` (using the `choice_value` type option), which then subsequently calls the Symfony Intl components
- Those `choices` are then sorted using `usort` which, for the comparison, calls... `Country->getName()` 
- Multiply all this process by something like 150 members on a single page...

Somehow all those forms shouldn't access to the Intl component data a trillion time, it should be passed by parent form (in those cases, the country list is not gonna change from one member to another). 

I fear this kind of thing could happen with every other Intl components (locales, currencies, etc...) but I don't think Sylius has another page which can potentially gather so many items of those types.

Any thoughts on this on the Sylius side ? :D

**Comment by stloyd** [comment]
@pamil @lchrusciel This issue was fixed in `master/1.8` with #10565.

**Comment by lchrusciel** [comment]
hey @stloyd, thanks for picking it up!
